client/components/mma/shared/ProgressIndicator.stories.tsx

```diff
@@ -10,37 +10,6 @@ export default {
 	},
 } as Meta<typeof ProgressStepper>;
```

Comment:
👀  Should this `ProgressStepper` reference still be here?

Reply:
ooo, no it shouldn't!